lib/bolt/config.rb

```diff
@@ -340,7 +340,16 @@ def deep_clone
     private def normalize_log(target)
       return target if target == 'console'
       target = target[5..-1] if target.start_with?('file:')
-      'file:' + File.expand_path(target, @project.path)
+      file = File.expand_path(target, @project.path)
+      begin
+        # It's easier to do this here and ignore the default log than to rewalk
+        # config and get the filenames.
+        FileUtils.touch(file) unless target == 'bolt-debug.log'
```

Comment:
We already fail in `Bolt::Logger` if a log file isn't writeable. So I think what we actually want is to try to touch bolt-debug.log and _remove_ it from the list of logs if that fails. Then we can let `Bolt::Logger` handle the others later in the process.